src/utils/type.utils.ts

```diff
@@ -87,3 +87,18 @@ export const isNull = (value: any): value is null => {
 export const isFunction = (value: any): value is Function => {
   return typeof value === 'function';
 };
+
+/**
+ * Type check for array of numbers
+ *
+ * @param {any} value Any value
+ * @return {boolean} True if it is a array of numbers, false otherwise
+ */
+export const isArrayOfNumbers = (value: any): value is boolean => {
```

Comment:
```suggestion
export const isArrayOfNumbers = (value: any): value is number[] => {
```

In typeguards, this param should return the correct type. Typeguards return a boolean by default + this is for the Typescript to be able to know what the type is after typeguard returns true.